### ISSUE TITLE
Pink point not appearing in pyramid (rope solo gets higher CPR?)

### ISSUE THREAD

**Issue by oadams** [theCrag/website#3921]
**What happened?**

I pink-pointed a climb at a given grade, but it's not seen in the pyramid to the side. See _Christian Crack_ under Notable Ascents in the screenshot below

Example url(s) to reproduce the problem:

https://www.thecrag.com/climber/oliveradams

Screenshot:

![screenshot](https://i.ibb.co/dpR3mjR/Screen-Shot-2021-05-23-at-3-28-41-PM.png)

**What you expected:**

* Some pink in the grade 20 bar in the pyramid.

I don't understand the mechanics at play here, but if I had to guess it's because toprope solos surprisingly have a smaller tick shift than pink points. In this case it's 100 for the TR solo and 130 for the pink point. When I click on the grade 20 bar in the pyramid, I do see _Christian Crack_, however it's listed as TR solo. Presumably this is because the CPR from the TR solo is higher. So maybe the real issue here is that you get more points for a TR solo than a pink point, which doesn't make sense since you don't have to spend energy clipping and the psychological element of fear of falling isn't there. For any reasonable TR solo setup, it is far easier than pink-pointing (even without weighting the rope).

Taking a look at the ![documentation on tick shift calculations](https://www.thecrag.com/en/article/tickshift) it seems the tick shifts are calculated via some statistical method, and the docs do reference some oddities regarding trad pink points. I would be interested to know what statistical approach is used to determine tick shits and maybe think about how we could bake in some prior information to the model so that the tick shifts at least observe some sort of reasonable ordering across tick types, with pink points being considered harder than rope solos.

**Comment by DaneEvans** [comment]
Is a trad PP even trad?

I'll hear the argument if the gear is poor, but generally I can't see it

**Comment by rouletout** [comment]
A pink point of a trad route is considered a sport ascent in CPR (see here: https://www.thecrag.com/en/article/ticktypes)

**Comment by oadams** [comment]
> A pink point of a trad route is considered a sport ascent in CPR (see here: https://www.thecrag.com/en/article/ticktypes)

Yes, it counts to sport CPR, but trad pink points appear in the trad pyramid. It would be reasonable behavior to put them in the sport pyramid, the problem here is inconsistency. As you can see, there is pink elsewhere in the trad pyramid.

**Comment by rouletout** [comment]
Agree, this seems less than optimal.
